jsHelper/spicetifyWrapper.js

```diff
@@ -439,6 +447,26 @@ window.Spicetify = {
 
 	if (Spicetify.Color) Spicetify.Color.CSSFormat = modules.find(m => m?.RGBA);
 
+	// Combine snackbar and notification
+	(async function bindShowNotification() {
+		if (!(Spicetify.Snackbar || Spicetify.sendNotification)) {
+			setTimeout(bindShowNotification, 10);
+			return;
+		}
+
+		Spicetify.showNotification = (message, isError = false, msTimeout) => {
+			if (!Spicetify.Snackbar?.enqueueSnackbar) {
+				Spicetify.notification(message, isError, msTimeout);
```

Comment:
I forgot to change it here
```suggestion
				Spicetify.sendNotification(message, isError, msTimeout);
```